### ISSUE TITLE
Question about how load order is determined

### ISSUE THREAD

**Issue by Victaria90** [shakeyourbunny/RWMSDB#39]
Is there a list for how you have specific mods set to load? 
I would love to be able to use it to self rearrange mods that aren't sorted yet. I have been using a sorting priority I got from the Jecrell discord but it is dated and not as categorized.

Here is the list i have been using
---From top to bottom:
--Core, Hugslib, Jecstools, Humanoid, Prepare Carefully (PC could possibly also go at the bottom of this list, it also has various issues that require arcane methods to get around)
Core includes libraries, if something says it has no content, and just has code for other mods, that is a library
THEN
Giddy Up and all expansions of such should go right below Humanoid, and then Doors, then Third Age
If you are running Combat Extended, it is theorized that it should go right above Third Age, and the patch to both goes right below both
AFTER ALL OF THAT
-Map Generation:
Anything that adds biomes, messes with how tiles and maps work, or changes how the world map generates
-Factions:
Anything that ONLY adds new factions, not new races or types of pawns. Faction Control by Rainbeau also goes here.
-Trait Mods:
Rainbeau has made a few of these, for example. If it just adds traits, or makes it possible to have more, it goes here.
-Medical Mods:
Yes this includes those that add new bionic parts. No it does not include mods that ONLY add medical items. EPOE and Rah's are examples of medical mods.
-Game Behavior:
While You're Up and Hand Me That Brick go here, but so do things like Steel Isn't Flammable and Fewer Inferno Cannons
Basically it's anything that changes how pawns, items, buildings, or events behave. Emphasis on "GAME" behavior.
-Items:
Clothing, hair, buildings, and items; hats and weapons and Vegetable Garden and giant nuclear power plants go here.
-Races (and new animals):
RACES GO AT THE BOTTOM THIS IS NOT A DRILL. This also includes Arachnophobia, Werewolves, Vampires, and other mods that make massive changes to pawns just by being loaded.

**Comment by shakeyourbunny** [comment]
RWMS uses just as any other similar tool a numeric sorting priority, but not directly to avoid some drawbacks. It is defined in rwms_db_categories.json in this project (and only changeable by the project owner). 

Its prioritization of the categories is similar but more fine-grained and has some different opinions other than than the basic library/framework stuff, especially that races are not at the bottom of the list (user interface, textures, "annoyances"/cheat stuff, patches are there). In addition there are some sub-categories there, that are not patches (like interface addons; eg interface: RimThemes, interface addons: theme (packs) for RimTheme).

Most annoying addon is imho Combat Extended which needs to have patches for basically everything, because it discards the normal combat system and Giddy Up which introduces many sub addons which are delicate to put in the right categories, but the latter is 100% done. 

I already did include your additions to the database this day early ago, but forgot to push them (which is done).
